### ISSUE TITLE
Which TLDR to keep? 

### ISSUE THREAD

**Issue by erinmr** [FAIRIslandProject/Generic-Place-based-Data-Policy#27]
https://github.com/FAIRIslandProject/Generic-Place-based-Data-Policy/blob/80783c9c67e2f7a8c7dd3cdee7e78949bdccb695/GenericPlacebasedDataPolicy.Rmd#L30

**Comment by erinmr** [comment]
Suggest using the TLDR from Tetiaroa draft policy with changes we've made to include institutional policy and remove elsewhere

 Tl;dr In return for the support and access to [Place] (the physical island and data derived from it) provided by [Host Organization], participants in research projects agree to make their metadata and prepublication data available to each other through a “[Place] Data Trust” that facilitates data re-use according to mutually agreed terms and a code of conduct. The Trust is maintained by the [Host Organization] to support decision-making for wise and equitable stewardship of the [place]. It follows best-practice in research data stewardship. This policy does not impose legally-binding conditions nor replace or impinge on legal obligations and/or institutional policies that might apply.

**Comment by erinmr** [comment]
Shorten the first sentence
